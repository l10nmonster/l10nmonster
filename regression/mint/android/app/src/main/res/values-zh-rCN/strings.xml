<?xml version="1.0" encoding="utf-8"?>
<resources>
    <string name="winter">冬天就&lt;b>要来了&lt;/b></string>
</resources>