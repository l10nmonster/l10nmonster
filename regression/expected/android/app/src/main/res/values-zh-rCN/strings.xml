<?xml version="1.0" encoding="utf-8"?>
<resources>
	<string name="winter">冬天就&lt;b>要来了&lt;/b></string>
	<string name="house">房子: %1$s 座右铭: %2$s</string>
	<string name="nothing">I know 100%%</string>
    <string name="everything">I know<strong><font color="%1$s">%2$s</font>%%</strong></string>
    <string name="frozen">Translation-Frozen: <strike>%1$s</strike></string>
	<string name="nbsp">Translation-The translation has a nbsp, source doesn't</string>
</resources>
